src/read.c

```diff
@@ -1284,21 +1289,20 @@ static avifBool avifDecoderGenerateImageGridTiles(avifDecoder * decoder, avifIma
 // dstImage. Only the freshly decoded tiles are considered, skipping the already
 // copied or not-yet-decoded tiles.
 static avifBool avifDecoderDataFillImageGrid(avifDecoderData * data,
-                                             avifImageGrid * grid,
                                              avifImage * dstImage,
-                                             unsigned int firstTileIndex,
+                                             const avifItemTiles * group,
```

Comment:
Nit: `group` => `itemTiles` or `imageTiles` or `tiles`

Also fix other occurrences.

Reply:
Done. At first I avoided that because it makes the order `tiles->grid` look weird. I also considered `item->grid` but it conflicts with `avifDecoderItem`.

Reply:
I am also not very happy with `tiles`. It's hard to come up with a good, short name for this parameter.

Reply:
One possibility is to name the struct `avifTileInfo` or `avifItemTileInfo`, and name the function parameter `info`.